exec/java-exec/src/main/java/org/apache/drill/exec/store/httpd/HttpdLogFormatPlugin.java

```diff
@@ -169,11 +119,15 @@ public HttpdLogRecordReader(final FragmentContext context, final DrillFileSystem
      * @return Map with Drill field names as a key and Parser Field names as a value
      */
     private Map<String, String> makeParserFields() {
-      final Map<String, String> fieldMapping = Maps.newHashMap();
+      Map<String, String> fieldMapping = Maps.newHashMap();
```

Comment:
According to [Maps docs](https://google.github.io/guava/releases/23.0/api/docs/com/google/common/collect/Maps.html#newHashMap--) `newHashMap()` ` [...] method is now unnecessary and should be treated as deprecated. Instead, use the HashMap constructor directly, taking advantage of the new "diamond" syntax.`

Reply:
Fixed